src/gui/input.cpp

```diff
@@ -195,6 +195,13 @@ QString convertKey(const QKeyEvent& ev) noexcept
 		return ToKeyString(GetModifierPrefix(mod), "Bslash");
 	}
 
+	// Issue#833: Handle MacOS Keyboard
+#ifdef __APPLE__
```

Comment:
No `#ifdefs` please :)

It makes the code more difficult to test. Avoiding `#ifdefs` allows us to test Windows/Linux/Mac behavior on one machine.

Instead, you can define a function in `input.h`, and implement it for each platform in `src/gui/input_{platform}.cpp`.

Alternatively, it may be easier to do platform specific fixup in `CreatePlatformNormalizedKeyEvent(...)`.